kadi/commands/commands.py

```diff
@@ -1,17 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 import tables
 import tables3_api
+from pathlib import Path
```

Comment:
I think kadi in general is still Python 2 compatible?  And I've forgotten the transition plan and/or requirements.

Reply:
Good point.  So this will drive the first 4.x version of kadi.  This change will not change the backing django database so that the current Py2 compatible 3.x version will continue working.

About the kadi web server, that's a big can of worms that I need to reopen.  IIRC the transition plans and requirements have not been formulated or written down, so you did not forget them.

Reply:
For kadi versioning, was your plan 4.17 or some such?